app/views/diggers/_from.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <%= simple_form_for(digger) do |f| %>
+  <%= f.input :name %>
+  <%= f.input :technical_specification %>
+  <%= f.input :start_date %>
+  <%= f.input :end_date %>
+  <%= f.input :day_rate %>
+  <%= f.input :photo %>
```

Comment:
why is this photo upload different to the one in the two other forms?

Reply:
i do not use this form

Reply:
then delete it